SU-SWS/decanter
# Fonts and chipping away at more base styles.

# READY FOR REVIEW 

# Summary
- Form elements and fonts

# Needed By (Date)
- Whenever

# Steps to Test

1. Copy sample.test.html into development environment
2. Start the development server and review the form elements
3. Ensure the fonts are loading.
